knowledge_base/share_files_across_bundles/shared/config/data.json

```diff
@@ -0,0 +1 @@
+["Lorem", "Ipsum", "Dolor", "Sit", "Amet"]
```

Comment:
Maybe you could show some shared `variables` instead, as seen in https://databricks.slack.com/archives/C074E9F4YG6/p1724420310720509?thread_ts=1724332254.479149&cid=C074E9F4YG6?

Reply:
Sorry, don't get the reference. Can you clarify?

Reply:
Discussed elsewhere: this will be a separate example.